systemtest/src/main/java/io/strimzi/systemtest/resources/crd/kafkaclients/AdminClientOperations.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.resources.crd.kafkaclients;
+
+public enum AdminClientOperations {
+    CREATE_TOPICS("create"),
```

Comment:
I think better would be just `CREATE`, `LIST` ...
```suggestion
    CREATE("create"),
```
Or we will use also ACLs operation?

Reply:
Originally it was like you suggest, later is was changed to X_TOPICS for possible future usage.